spring-pulsar/src/main/java/org/springframework/pulsar/core/reactive/ReactivePulsarSenderOperations.java

```diff
@@ -45,6 +47,24 @@
 	 */
 	Mono<MessageId> send(String topic, T message);
 
+	/**
+	 * Sends multiple messages to the default topic in a reactive manner.
+	 * @param messages the messages to send
+	 * @return the ids assigned by the broker to the published messages in the same order
+	 * as they were sent
+	 */
+	Flux<MessageId> send(Publisher<T> messages);
```

Comment:
What is the benefit of accepting publisher as input? 

I do understand that `Publisher` is the Reactivestreams API rather than Reactor (aka more generic). I wonder if mixing them may be a bit confusing? Or is this a common practice in other Reactive frameworks?

Reply:
Yes that's common practice.
See for instance in [R2DBC](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/reactive/ReactiveCrudRepository.html#saveAll-org.reactivestreams.Publisher-) or [Spring-Kafka](https://docs.spring.io/spring-kafka/api/org/springframework/kafka/core/reactive/ReactiveKafkaProducerTemplate.html#send(org.reactivestreams.Publisher)).
This allows the user to more easily mix between libs that may not use the same reactive streams impl.
A good application of the open/close principle / Postel law 😃

Reply:
We're also doing it in the Pulsar reactive client : https://github.com/apache/pulsar-client-reactive/pull/12